### ISSUE TITLE
[Meeting] 01/16/2023

### ISSUE THREAD

**Issue by yehyunsuh** [yehyunsuh/Total-Knee-Replacement-Post-Surgical-Assessment#15]
## What
Not showing any learning in our model
<br>

## Why
Tried to define multiple reasons
<br>

## How
- [x] Dilate to 40(or even bigger) to see if the model can overfit or segment the circle
- [x] Plot results in a picture with white/black
- [x] Use optimizer.step() instead of scaler
- [x] No augmentation
- [x] Sigmoid when using model from scratch

**Comment by yehyunsuh** [comment]
Issue closed after #16
